ux/build.gradle

```
@@ -6,8 +6,8 @@ android {
         // Sceneform requires minSdkVersion >= 24.
         minSdkVersion 24
         targetSdkVersion 30
-        versionCode 9
-        versionName "1.18.11"
+        versionCode 10
+        versionName "1.18.12"
```

I will make the release with one more thing just after you
```suggestion
        versionName "1.18.11"
```